crates/typst-library/src/visualize/polygon.rs

```diff
@@ -19,13 +23,15 @@ use crate::prelude::*;
 /// Display: Polygon
 /// Category: visualize
 #[element(Layout)]
+#[scope(scope.define("regular", regular_polygon_func()); scope)]
```

Comment:
For consistency with other function scope declarations

```suggestion
#[scope(
    scope.define("regular", regular_polygon_func());
    scope
)]
```